open_samus_returns_rando/files/randomizer_powerup.lua

```diff
@@ -32,7 +35,7 @@ function RandomizerPowerup.OnPickedUp(actor, resources)
     RandomizerPowerup.ChangeSuit()
 
     for _, resource in ipairs(granted) do
-        RandomizerPowerup.IncreaseEnergy(resource)
+        RandomizerPowerup.IncreaseAmmo(resource)
```

Comment:
There is no `IncreaseAmmo` function in `RandomizerPowerup`. This will throw a lua error.
Very likely that it still works as there is the function is done after this but not really a nice thing.